config/initializers/subscriptions.rb

```diff
@@ -45,12 +45,21 @@
   Notifier.subscribe(event_name: Notifier::REVIEW_REQUESTED) do |payload|
     ReviewsMailer.with(**payload).submitted_email.deliver_later
   end
+  Notifier.subscribe(event_name: Notifier::REVIEW_REQUESTED) do |payload|
+    ReviewRequestedEventSubmitter.call(**payload)
+  end
```

Comment:
Just to confirm: it's preferable to create identical subscriptions that each handle a single concern rather than to create a single subscription that handles multiple concerns? If so, we may want to add a comment somewhere in here.

Reply:
I'm not aware of any reason to prefer one over the other. There was already examples of identical subscriptions, so I continued that convention.